### DISCUSSION TITLE
Compatible with Intex PureSpa #28408?

### DISCUSSION THREAD

**Discussion by santiagos5** [Yogui79/IntexPureSpa#28] [Q&A] [answered]
Hi, is this project compatible with Intex PureSpa #28408? I think the SPA version is 2018 or 2019

**Comment by djgel** [comment, marked as answer]
28408 is not Wireless  control... the system was created making a 2.4Ghz signal interption, sending and getting the information! With this not being compatible with your model! To automate yours, you can put an ESP turned on on the control board, eg "https://www.youtube.com/watch?v=Uu1ymWx3O58" and "https://github.com/UlrichMai/MaiPureSpaController"

**Reply by santiagos5** [reply]
Thanks
